tailwind.js

```diff
@@ -48,6 +48,7 @@ let colors = {
 
   'black': '#202939',
   'white': '#ffffff',
+  'blue': '#1f386e',
```

Comment:
Please use a different name for this color, maybe `neutral`, later on we might want to add some variations.